br/tests/utils.go

```diff
@@ -0,0 +1,207 @@
+// Copyright 2024 PingCAP, Inc. Licensed under Apache-2.0.
+
+package main
+
+import (
+	"encoding/binary"
+	"flag"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/klauspost/compress/zstd"
+)
+
+const (
+	cmdValidateBackupFiles = "validateBackupFiles"
+	extSST                 = ".sst"
+	extLOG                 = ".log"
+)
+
+func main() {
+	if len(os.Args) < 2 {
```

Comment:
You may use [`cobra.Command`](https://pkg.go.dev/github.com/spf13/cobra#Command) instead of manually parse the command.

Reply:
gotcha